pkgs/applications/editors/emacs-modes/org-packages.nix

```diff
@@ -0,0 +1,42 @@
+/*
+
+# Updating
+
+To update the list of packages from ELPA,
+
+1. Clone https://github.com/ttuegel/emacs2nix
+2. Run `./org-packages.sh` from emacs2nix
+3. Copy the new elpa-packages.json file into Nixpkgs
+4. `git commit -m "elpa-packages $(date -Idate)"`
+
+*/
+
+{ fetchurl, lib, stdenv, texinfo }:
+
+self:
+
+  let
+
+    imported = import ./org-generated.nix {
+      inherit (self) callPackage;
+    };
+
+    super = imported;
+
+    elpaBuild = import ../../../build-support/emacs/elpa.nix {
+      inherit fetchurl lib stdenv texinfo;
+      inherit (self) emacs;
+    };
+
+    markBroken = pkg: pkg.override {
+      elpaBuild = args: self.elpaBuild (args // {
+        meta = (args.meta or {}) // { broken = true; };
+      });
+    };
+
+    overrides = {
+    };
+
+    orgPackages = super // overrides;
+
+  in orgPackages // { inherit elpaBuild orgPackages; }
```

Comment:
@mdorman It shouldn't be necessary to `inherit elpaBuild` here. The packages which require it will get it from `elpaPackages`. It probably won't hurt anything for the moment, but I worry it could get confusing later to have `elpaBuild` defined in multiple places (since this definition overrides the first).